crates/sp-domains/src/lib.rs

```diff
@@ -222,14 +219,8 @@ pub struct ProofOfElection<DomainHash> {
     pub operator_public_key: OperatorPublicKey,
     /// Global challenge.
     pub global_challenge: Blake2b256Hash,
-    /// Storage proof containing the partial state for verifying the bundle election.
-    pub storage_proof: StorageProof,
-    /// State root corresponding to the storage proof above.
-    pub system_state_root: DomainHash,
-    /// Number of the system domain block at which the proof of election was created.
-    pub system_block_number: BlockNumber,
-    /// Block hash corresponding to the `block_number` above.
-    pub system_block_hash: DomainHash,
```

Comment:
Does this means we will use the current epoch's staking summary of the domain for verifying the `ProofOfElection`?

Reply:
> Does this means we will use the current epoch's staking summary of the domain for verifying the ProofOfElection in the consensus chain runtime while submitting bundle?

I thought that was what we wanted to do. Is that not the case?

> Two general ideas, using the consensus chain state in the runtime, or generating a storage proof and including it in the proof of election, need more time to assess them.

Can you expand more on this? Is that not a decision we do not for this PR ?